src/YoutubeAtom.test.tsx

```diff
@@ -19,9 +19,5 @@ describe('YoutubeAtom', () => {
         const { getByTitle } = render(atom);
 
         expect(getByTitle('My Youtube video!')).toBeInTheDocument();
-        expect(getByTitle('My Youtube video!')).toHaveAttribute(
-            'src',
-            expect.stringMatching(/https:\/\/www.youtube.com\/embed.*/),
-        );
```

Comment:
Is there a way we can keep this test, or add another few to make sure this functions as expected?

Reply:
not blocking merge but completely agree